### DISCUSSION TITLE
Default commands

### DISCUSSION THREAD

**Discussion by strokirk** [nvim-neotest/neotest#227] [Ideas]
Hi! 

It seems that most of the API for this plugin is lua-only, which makes it a bit hard to get started. Would it make sense to add a bunch of `:Neotest` commands to make it easy to get started?

**Comment by rcarriga** [comment]
I'm not opposed to the idea but I'm not particularly interested in this feature myself and have little experience with writing commands with argument parsing/completion so can't say it'll be a priority. Would be happy to help with a PR! :smile:

**Reply by MunifTanjim** [reply]
- https://github.com/nvim-neotest/neotest/pull/254
